### PR TITLE
Add data-source configurations for EI Analytics

### PR DESCRIPTION
**Pull request by isharailanga**
## Purpose
To add EI_ANALTICS datasource configurations required by Enterprise Integrator Analytics Feature.

### CODE REVIEW

File: modules/distribution/carbon-home/conf/dashboard/deployment.yaml [hunk 1]
```diff
@@ -366,6 +366,23 @@ wso2.datasources:
         validationTimeout: 30000
         isAutoCommit: false
 
+  - name: EI_ANALYTICS
+    description: "The datasource used for EI Analytics dashboard feature"
```
Comment (Niveathika92) [inline modules/distribution/carbon-home/conf/dashboard/deployment.yaml]: used to store EI analytics statistics

File: modules/distribution/carbon-home/conf/dashboard/deployment.yaml [hunk 2]
```diff
@@ -366,6 +366,23 @@ wso2.datasources:
...
+    jndiConfig:
+      name: jdbc/EI_ANALYTICS
+    definition:
+      type: RDBMS
+      configuration:
+        jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/dashboard/database/EI_ANALYTICS;AUTO_SERVER=TRUE'
```
Comment (Niveathika92) [inline modules/distribution/carbon-home/conf/dashboard/deployment.yaml]: Lets create the database in worker profile as a standard. jdbc:h2:${sys:carbon.home}/wso2/worker/database/EI_ANALYTICS;AUTO_SERVER=TRUE

**Comment by isharailanga** [comment]
Fixed in 97933ae0a8e13300895900efb121821109608a10

File: modules/distribution/carbon-home/conf/dashboard/deployment.yaml (hunk 2 above)
Comment (isharailanga) [reply modules/distribution/carbon-home/conf/dashboard/deployment.yaml]: Fixed in 97933ae0a8e13300895900efb121821109608a10

File: modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java [hunk 3]
```diff
@@ -100,11 +100,11 @@
 )
 public class CalculatePerformanceStreamProcessorExtension extends StreamProcessor {
```
Comment (Niveathika92) [inline modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java]: Is this a merge commit? Why are these changes committed?

**Pull request closed without merging**